configs/registry/repository-volume.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: docker-repo-pv
+  labels:
+    usage: registry
+spec:
+  capacity:
+    storage: 5Gi # TODO: size
```

Comment:
as for lines 9 and 25 (i.e., storage size), please use an environment variable from the setup_system.sh script. 5GB is a fine constant to me. 
Please take a look at how we set up MinIO storage size dynamically [here](https://github.com/ease-lab/vhive/blob/master/docs/developers_guide.md#minio-s3-service)